drivers/hmis/spec/requests/hmis/update_external_form_submission_spec.rb

```diff
@@ -0,0 +1,124 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'rails_helper'
+require_relative '../hmis/login_and_permissions'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.describe 'Update External Form Submission', type: :request do
+  include_context 'hmis base setup'
+
+  subject(:mutation) do
+    <<~GRAPHQL
+      mutation UpdateExternalFormSubmission(
+        $id: ID!
+        $projectId: ID!
+        $input: ExternalFormSubmissionInput!
+      ) {
+        updateExternalFormSubmission(id: $id, projectId: $projectId, input: $input) {
+          externalFormSubmission {
+            id
+            submittedAt
+            spam
+            status
+            notes
+            definition {
+              identifier
+            }
+            values
+          }
+          #{error_fields}
+        }
+      }
+    GRAPHQL
+  end
+
+  let!(:access_control) do
+    create_access_control(hmis_user, p1, with_permission: [:can_manage_external_form_submissions, :can_view_project])
+  end
+
+  before(:each) do
+    hmis_login(user)
+  end
+
+  context 'when reviewing a submission' do
+    let!(:definition) { create(:hmis_external_form_definition) }
+    let!(:submission) { create(:hmis_external_form_submission, definition: definition, submitted_at: Date.current) }
```

Comment:
Wondering why pass `submitted_at: Date.current`? (The factory default is `Time.current`, and its a datetime field)

Reply:
These can be removed